src/polynomial.rs

```diff
@@ -127,13 +126,15 @@ impl fmt::Display for Polynomial {
 }
 
 #[cfg(feature = "python")]
-#[cfg_attr(feature = "python", pymethods)]
+#[pymethods]
```

Comment:
The feature=python cfg is now duplicated.

Reply:
that does not work. I did not do that for no reasons (as always).
Currently it's back to original state, so you will either have to revert the latest commit or figure out what is wrong